sample/app/src/App.jsx

```diff
@@ -1,23 +1,55 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
 import './App.css';
 
 const App = () => {
-  let [isSignedIn, setIsSignedIn] = React.useState(false)
+  let [isSignedIn, setIsSignedIn] = React.useState(false);
+  
   return (
-    <div className="App">
-      {!isSignedIn &&
-        <button onClick={() => setIsSignedIn(true)}>
-          SIGN IN
-        </button>
-      }
-
-      {isSignedIn &&
-        <button onClick={() => setIsSignedIn(false)}>
-          LOG OUT
-        </button>
-      }
-    </div>
+    <Router>
+      <div className="App">
+        <div className="header">
+          <nav>
+            <div className="label">
+              Nav
+            </div>
+            <Link to="/">/home</Link>
+            <Link to="/about">/about</Link>
+          </nav>
+          <div className="button">
+            <div className="label">
+              Button
+            </div>
+            {!isSignedIn &&
+              <button onClick={() => setIsSignedIn(true)}>
+                SIGN IN
+              </button>
+            }
+            {isSignedIn &&
+              <button onClick={() => setIsSignedIn(false)}>
+                LOG OUT
+              </button>
+            }
+          </div>
+        </div>
+        <div className="body">
```

Comment:
```suggestion
        <main className="body">
```

For semantics, I think this and the closing div tag could become `main`

Reply:
👍 I also replaced `<div className="header">` with `<header>`